packages/esm-patient-chart-app/src/view-components/chart-review.component.tsx

```diff
@@ -46,32 +68,20 @@ const ChartReview: React.FC<ChartReviewProps> = ({ patientUuid, patient, view, s
         })}
       />
     );
-  } else if (dashboard.config.type === 'grid') {
+  } else {
     return (
       <>
         {dashboard.title && <h1 className={styles.dashboardTitle}>{dashboard.title}</h1>}
-        <GridView
-          slot={dashboard.slot}
-          layout={dashboard.config}
-          name={dashboard.name}
-          patient={patient}
-          patientUuid={patientUuid}
-        />
+        <ExtensionSlot
+          key={dashboard.slot}
+          extensionSlotName={dashboard.slot}
+          className={styles.dashboard}
+          style={{ gridTemplateColumns }}
+        >
+          <Extension state={state} wrap={wrapItem} />
```

Comment:
This seems weird to be passing patient and patientUuid through as props to ChartReview when ChartReview could just be using usePatient(), then even weirder still that we memoize it and pass it through to Extension. Why is this not something that Extension handles natively? And if it's not something that should be in Extension then the end components should be more responsible for calling usePatient() rather than passing props so deep?

Reply:
Yeah, I think it is bad that we pass `patient` as a param. You are right that that should be obtained with `usePatient`. The reason we pass `patientUuid` as a param is so that widgets can be used in contexts where there is information for multiple patients displayed in the same page.